lib/archethic/mining/transaction_context.ex

```diff
@@ -58,9 +58,11 @@ defmodule Archethic.Mining.TransactionContext do
       |> Enum.uniq()
 
     prev_tx_task = request_previous_tx(previous_address, prev_tx_nodes_split)
+    utxos_task = request_utxos(previous_address, unspent_outputs_nodes_split)
     nodes_view_task = request_nodes_view(node_public_keys)
 
     prev_tx = Task.await(prev_tx_task)
+    utxos = Task.await(utxos_task)
```

Comment:
You could probably use [Task.await_many](https://hexdocs.pm/elixir/Task.html#await_many/2)

Reply:
Done !